GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -51,9 +51,6 @@ typedef NS_ERROR_ENUM(kGIDSignInErrorDomain, GIDSignInErrorCode) {
   kGIDSignInErrorCodeMismatchWithCurrentUser = -9,
 };
 
-/// Represents a completion block that takes an error if the operation was unsuccessful.
-typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
-
```

Comment:
Maybe we can keep this in `GIDSignIn_Private.h`?

Reply:
Sure, this would be consistent with what we've done for `GIDUserAuthCompletion`.  Added to private header.